packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -11,16 +11,16 @@ example](https://docs.aws.amazon.com/step-functions/latest/dg/job-status-poller-
 const submitLambda = new lambda.Function(this, 'SubmitLambda', { ... });
 const getStatusLambda = new lambda.Function(this, 'CheckLambda', { ... });
 
-const submitJob = new stepfunctions.Task(this, 'Submit Job', {
-    resource: submitLambda,
+const submitJob = new lambda.FunctionTask(this, 'Submit Job', {
+    function: submitLambda,
```

Comment:
`function` might be reserved... use `handler`?

Reply:
Doh. Of course.